### DISCUSSION TITLE
New firmware for RLC-542WA with iframe-Settings

### DISCUSSION THREAD

**Discussion by Se7enair** [AT0myks/reolink-fw-archive#14] [Firmware sharing]
I aksed the upport for a firmware with iframe settings. They sent me a new firmware.
I can confirm that it is working for me.

[IPC_51316M.757_21121503.RLC-410W.ov4689.4MP.WIFI1021.REOLINK.zip](https://github.com/AT0myks/reolink-fw-archive/files/11366606/IPC_51316M.757_21121503.RLC-410W.ov4689.4MP.WIFI1021.REOLINK.zip)

**Comment by AT0myks** [comment]
Thanks, the firmware you attached has already been added after #12 and is for the RLC-410W. But your title says RLC-542WA, maybe you attached the wrong file?

**Comment by Se7enair** [comment]
Damn, wrong file....
Correct file  https://we.tl/t-cgHeBV6CLp

**Reply by AT0myks** [reply]
Is this the link they sent you via email, and if it's not and you still have it could you share the one they sent you?

**Reply by Se7enair** [reply]
They send it as a attachement.

**Reply by AT0myks** [reply]
Thanks for confirming and sharing. I added the firmware.

**Reply by CarminKern** [reply]
Hello I also need the new firmware, unfortunately the link has expired

**Comment by CarminKern** [comment]
Hello I also need the new firmware, unfortunately the link has expired.

**Reply by Se7enair** [reply]
Search here https://github.com/AT0myks/reolink-fw-archive , there are all firmwares listed

**Reply by CarminKern** [reply]
Version v3.1.0.1983_23040623  is Linked to this

**Reply by Se7enair** [reply]
Click on the version on the left. Not on source.

**Reply by CarminKern** [reply]
Oh... Thanks!!!
